website/pages/en/index.js

```diff
@@ -464,7 +351,6 @@ class Index extends React.Component {
           <TldrSection language={language} />
           <LanguagesSection />
           <EditorSupportSection />
-          <GetStartedSection />
```

Comment:
Removing this breaks the light-dark alternating background colors. Changing the “Used By People You Rely On” to have the lighter background would restore the alternation.